scripts/Utils/ProcessBadFilesList.py

```diff
@@ -0,0 +1,124 @@
+"""
+Peocesses files with info about corrupted/suspicious input files
+Gather statistics and optionally report to Rucio
+ref. https://github.com/dmwm/CRABServer/issues/7548#issuecomment-1714567989
+"""
+import os
+import json
+import datetime
+import shutil
+
+
+ERROR_KINDS = [
+    'not a ROOT file',
+    'truncated',
+    ]
+
+
+def main():
+    """" description is at line 1 """
+    # location of files prepared by RetryJobs
+    topDir = '/eos/cms/store/temp/user'
+    # topDir = '.'  # `for testing locally
+    # topDir = topDir + '/filechecker'  # currently not used`
+
+    for subDir in ['corrupted']:  # maybe later on add suspicious files, initially ignore them
+        totals = {}
+        doneFiles = []
+        myDir = f"{topDir}/{subDir}"
+        newFilesDir = myDir + '/new'
+        doneFilesDir = myDir + '/done'
+        newFiles = os.listdir(newFilesDir)
+        for aFile in newFiles:
+            filePath = f"{newFilesDir}/{aFile}"
+            result = parse(filePath)
+            accumulate(totals, result)
+            doneFiles.append(filePath)
+        print(totals)
+
+        for file in doneFiles:
+            shutil.move(file, doneFilesDir)
+
+        # now that we have the totals for this run of the script, update the
+        # daily tally
+        addToDailyLog(logDir=myDir, totals=totals)
```

Comment:
Should we move to a different dir for saving statistics? In case we mess up, we can purge the whole directory.

Reply:
yes. Current directory structure is not ideal already. I struggle to come up with a good name in `/store/temp/user/A-NAME/corrupted|suspicious|statistics`

Reply:
I picked : `/store/user/temp/BadInputFiles` as top directory for all stuff belonging this "area'